### ISSUE TITLE
[Update] Approve "Full DVD Entries"

### ISSUE THREAD

**Issue by AdultSun** [stashapp/StashDB-Docs#34]
Needs official approval:
https://guidelines.stashdb.org/docs/scenes/movies-dvds/#full-dvd-entries

Currently, we only have this section to explain how we handle **full movies** but we don't clarify our stance on **DVD/movie scenes** anywhere else. This should be covered in a separate section, but the two should still link/refer to each other in their descriptions.

**Comment by PhuriousGeorge** [comment]
Agreed there should be a dedicated section on DVD/Movie scenes; I've been fixing several of these scene titles, dates, etc

**Comment by AdultSun** [comment]
Updated language to better fit how full movies are handled on StashDB: #56

Still an unofficial guideline, pending formal approval.
